src/base/NekInterface.C

```diff
@@ -553,6 +553,56 @@ void flux(const int elem_id, const int order, double * flux_face)
   }
 }
 
+void velocity(const int elem_id, const int order, const double* velocity_SAM)
```

Comment:
The `velocity` and `temperature` functions can be a bit simpler here, since we're not doing any interpolation - you can remove `order` as an input parameter and delete a few unused local variables in the functions themselves (`end_1d` for instance).

Reply:
These functions are now located in `src/base/NekRSSeparateDomain.C` in the updated files. But I'm not sure which variables there aren't necessary for example specifically in `velocity` or in `sendBoundaryVelocityToNek`.